backend/chainlit/config.py

```diff
@@ -64,6 +64,9 @@
 # Link to your github repo. This will add a github button in the UI's header.
 # github = ""
 
+# Specify a CSS file that can be used to customize the user interface.
+# custom_css = '/assets/test.css'
```

Comment:
```suggestion
# custom_css = '/public/test.css'
```

I think most people will serve the css either from the public dir or an external link

Reply:
I have updated the pull request. Please check.